Instruments/source/CacheMerger.cxx

```diff
@@ -0,0 +1,92 @@
+/*! Merge multiple CacheTuples files into a single file.
+This file is part of https://github.com/hh-italian-group/h-tautau. */
+
+#include <iostream>
+
+#include "AnalysisTools/Core/include/RootExt.h"
+#include "AnalysisTools/Run/include/program_main.h"
+#include "h-tautau/Core/include/CacheTuple.h"
+#include "h-tautau/Core/include/EventTuple.h"
+#include "h-tautau/Analysis/include/EventCacheProvider.h"
+#include "h-tautau/Core/include/SummaryTuple.h"
+
+struct Arguments {
+    REQ_ARG(std::vector<std::string>, inputs);
+    REQ_ARG(std::string, outputFile);
+    REQ_ARG(std::string, channel);
+};
+
+namespace analysis {
+
+class CacheMerger {
+public:
+    CacheMerger(const Arguments& _args): args(_args), output(root_ext::CreateRootFile(args.outputFile())),
+                                         output_summary(ntuple::CreateSummaryTuple("summary", output.get(),
```

Comment:
cache tuples contain `cache_ntuple::CacheSummaryTuple` not `ntuple::SummaryTuple`